pyproject.toml

```diff
@@ -69,3 +69,6 @@ queries_path = "queries"
 target_package_name = "api"
 target_package_path = "src/greynoiselabs"
 convert_to_snake_case = "false"
+
+[tool.ruff]
+extend-exclude = ["src/greynoiselabs/api"]
```

Comment:
Just added this to the exclude given the client code is 100% generated and the line length and a few other things get flagged. Eventually we might not even check in the generated code.